packages/yourloops/lib/auth/hook.tsx

```diff
@@ -504,6 +504,14 @@ export function AuthContextImpl(api: AuthAPI): AuthContext {
     // return unmount;
   };
 
+  const certifyProfessionalAccount = async (): Promise<void> => {
+    console.log(historyHook.location.pathname);
+    if (historyHook.location.pathname === "/professional/certify") {
+      return;
+    }
+    window.location.href = await api.redirectToProfessionalAccountLogin();
```

Comment:
```suggestion
    window.location.href = `${window.config?.API_HOST}/auth/oauth/login`;
```
This will fix all your CORS issues :rocket: